scipy-stubs/sparse/_construct.pyi

```diff
@@ -7,6 +7,10 @@ import numpy.typing as npt
 import optype.numpy as onp
 import optype.numpy.compat as npc
 import optype.typing as opt
+from optype.numpy._any_dtype import f_cls
```

Comment:
if you `import optype as op`, then this private type alias can be replaced with `type[op.JustFloat]`